addons/interact_menu/XEH_clientInit.sqf

```diff
@@ -77,10 +77,24 @@ format ["%1 (%2)", (localize LSTRING(SelfInteractKey)), localize ELSTRING(common
 
 // background options
 ["ace_interactMenuOpened", {
-    if (GVAR(menuBackground)==1) then {[QGVAR(menuBackground), true] call EFUNC(common,blurScreen);};
-    if (GVAR(menuBackground)==2) then {0 cutRsc[QGVAR(menuBackground), "PLAIN", 1, false];};
+    params ["_menuType"];
+    if (_menuType == 0) exitWith {
```

Comment:
I would remove code duplicating here and below this way (not tested, just concept):
```
private _menuBackgroundSetting = [GVAR(menuBackground), GVAR(menuBackgroundSelf)] select _menuType;
if (_menuBackgroundSetting == 1) exitWith {[QGVAR(menuBackground), true] call EFUNC(common,blurScreen);};
if (_menuBackgroundSetting == 2) exitWith {0 cutRsc [QGVAR(menuBackground), "PLAIN", 1, false];};
```
Note `exitWith` instead of `then`, spaces in syntax and `QGVAR(menuBackground)` for `cutRsc` in both cases which looks like have to be existing class: https://github.com/acemod/ACE3/blob/e5a15d200f44df5fccc0bc5575d18d80b35538dd/addons/interact_menu/CursorMenus.hpp#L9